perllib/FixMyStreet/App/Controller/Waste.pm

```diff
@@ -311,8 +311,14 @@ sub confirm_subscription : Private {
     if ($already_confirmed) {
         my $payment = $p->get_extra_field_value('payment');
         $payment = sprintf( '%.2f', $payment / 100 );
+        my $reference_text;
+        if (!$reference) {
+            $reference_text = 'paid by phone/cheque';
```

Comment:
Could this be "reference $chequeReference (phone/cheque)" (looking up the ref) so that it matches up at the `open311-adapter` end.